examples/hcloud-k3s/README.md

```diff
@@ -2,9 +2,27 @@
 
 Configuration in this directory creates a k3s cluster resources including network, subnet and instances.
 
+## Preparations
+
+Make sure your [SSH-Agent](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent) is running (it is neccessary for Terraform), and if not, start it and add your ssh-key:
```

Comment:
This is a good one. I always run into this one.

Reply:
But instead of writing an example wouldn't it be better to simply link to some official documentation?

Reply:
I thought the Link to SSH-Agent Docs would be enough. How would you do it? :)

Reply:
Sounds great with a documentation link, thanks a lot for your reviews !